### PR TITLE
Disable KillTrack in instances (FPS improve)

### PR DESCRIPTION
**Pull request by vvv444**
Two new configuration options were added: "Disable in dungeons" and "Disable in raids".

### CODE REVIEW

File: Options.lua [hunk 1]
```diff
@@ -162,12 +162,28 @@ function Opt:Show()
         end)
     minimap:SetPoint("TOPLEFT", purge, "BOTTOMLEFT", 0, -8)
 
+    local disableDungeons = checkbox("Disable in dungeons (save CPU)",
+        "With this is checked, mob kills in dungeons won't be counted.",
```
Comment (Sharparam) [inline Options.lua]: With -> When

File: Options.lua [hunk 2]
```diff
@@ -162,12 +162,28 @@ function Opt:Show()
...
+        function(check, checked)
+            KT.Global.DISABLE_DUNGEONS = checked
+        end)
+    disableDungeons:SetPoint("TOPLEFT", minimap, "BOTTOMLEFT", 0, -8)
+
+    local disableRaids = checkbox("Disable in raids (save CPU)",
+        "With this is checked, mob kills in raids won't be counted.",
```
Comment (Sharparam) [inline Options.lua]: With -> When

File: Options.lua (hunk 1 above)
Comment (vvv444) [reply Options.lua]: Well, English is not my mother tongue... All existing options use "with" :-D
But no problem, I'll commit a fix :-)

**Comment by vvv444** [comment]
Thanks!

**Pull request merged**